app/javascript/components/search/genes/GeneSearchView.js

```diff
@@ -27,7 +27,7 @@ export default function GeneSearchView() {
                                  !geneSearchState.isLoading &&
                                  !geneSearchState.isError
 
-  let geneSearchPlaceholder = 'Search for genes across all studies'
+  let geneSearchPlaceholder = 'genes (e.g. "agpat pten")'
```

Comment:
Nit: title case is SCP and DSP convention.

```suggestion
  let geneSearchPlaceholder = 'Genes (e.g. "agpat pten")'
```